lib/bot.js

```diff
@@ -106,44 +88,47 @@ class Bot {
     return this._safelyRunApiCommand({method: 'read', options: {channel, peek, unreadOnly}})
   }
 
-  // --------------------------------------------------------------------------
-
   chatDelete(options: {channel: ChatChannel, messageId: number}): Promise<any> {
     const {channel, messageId} = options
     return this._safelyRunApiCommand({method: 'delete', options: {channel, messageId}})
   }
 
-  // --------------------------------------------------------------------------
-
   myInfo(): ?DeviceUsernamePair {
     return this._dPair
   }
 
-  // --------------------------------------------------------------------------
-
-  watchChannelForNewMessages(options: {channel: ChatChannel, onMessages: MessagesHandler}) {
-    const {channel, onMessages} = options
-    const key = chatChannelToKey(channel)
-    if (this._channelWatchers.has(key)) {
-      throw new Error(`already watching the channel ${JSON.stringify(channel)}`)
-    } else {
-      this._channelWatchers.set(key, new ChannelWatcher({channel, onMessages, bot: this}))
-    }
+  /**
+   * Listens for new chat messages on a specified channel. The `onMessage` function is called for every message your bot receives.
+   * @param onMessage
+   * @memberof Bot
+   */
```

Comment:
Just a quick heads up on the JSDOC stuff. I'm currently finishing the documentation build on `thebearjew/documentation-js`.

The approach is to move all the explanations & examples that _were_ in the README into JSDOC comments above each method. More than likely, these comments will be replaced in favor of the README comments.